src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 0.3s;
+
+    &:hover {
+      transition-duration: 0.3s;
```

Comment:
`transition-duration: 0.3s;`  already have this value in element, so its not necessary.

Reply:
Agree, fix it